packages/utils/src/dhis.js

```diff
@@ -41,6 +41,9 @@ const getServerName = (entityCode, isDataRegional) => {
     : REGIONAL_SERVER_NAME; // If the country does not have a dhis2 server, use the regional server
 };
 
+const getServerVariable = (serverName, variableName) =>
+    process.env[`${serverName.toUpperCase()}_${variableName}`] || process.env[variableName];
```

Comment:
Might be easier to just hard code it up by `SERVER_NAMES` given it's not going to change across deployments, so we can save ourselves from managing an environment variable?